config/nephe.yml

```diff
@@ -37,18 +37,13 @@ spec:
   - name: v1alpha1
     schema:
       openAPIV3Schema:
-        description: CloudEntitySelector is the Schema for the cloudentityselectors
```

Comment:
update your nephe builder image. it should now use kustomizer 4.4.1

Reply:
Done